src/main/kotlin/domain/game/LottoMachine.kt

```diff
@@ -7,15 +7,18 @@ import domain.money.Money
 import util.common.generateDistinctRandomNumbers
 
 class LottoMachine {
-    fun purchaseLottos(money: Money): List<PurchasedLotto> = mutableListOf<PurchasedLotto>().apply {
+    fun purchaseManualLottos(manualLottos: List<Lotto>): List<PurchasedLotto> =
+        manualLottos.toList().map { PurchasedLotto(it) }
```

Comment:
```suggestion
        manualLottos.map { PurchasedLotto(it) }
```
컬렉션과 관련된 함수들은(filter, map 등) 연산을 한 후에 원본이 아닌 복사본을 리턴하게 됩니다.